event/error.go

```diff
@@ -0,0 +1,18 @@
+package event
+
+type Error struct {
+	err     error
+	logData map[string]interface{}
+}
+
+func (e *Error) Error() string {
+	return e.err.Error()
+}
+
+func (e *Error) LogData() map[string]interface{} {
+	return e.logData
```

Comment:
This is actually by design, the callback functions designed for grabbing this data back check for nil and ignore it rather than printing empty log objects.

Reply:
In fact I've just realised that this package/PR is missing the interface definitions and callback functions so this error doesn't make sense as it is. I don't think it's even used yet either